### DISCUSSION TITLE
Permission denied on export?

### DISCUSSION THREAD

**Discussion by maximjsx** [outline/outline#7352] [Self Hosting]
I'm just wondering how this happend?

![image](https://github.com/user-attachments/assets/8bb7f317-e1f2-4082-904f-2e46b2353c41)

![image](https://github.com/user-attachments/assets/f401760e-1ea5-43c0-a094-09e30238e099)

Like im using docker for outline and it worked before.. I didn't changed anything and now out of nowhere the export fails...

So why can't it create a folder anymore there?

**Comment by maximjsx** [comment]
I literally didn't touched the server at all

**Reply by tommoor** [reply]
This would suggest that there is no longer permission to write to the directory on the host machine, you should check the permissions. Perhaps the host machine or docker volume ran out of disk space.

**Comment by maximjsx** [comment]
Also why can't I delete the failed export messages like the normal exports
![image](https://github.com/user-attachments/assets/4207c5aa-77e5-47d8-b728-f223e2c786fc)
